jenkins-pipelines/artifacts-oel81.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#! groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+artifactsPipeline(
+    test_config: 'test-cases/artifacts/oel81.yaml',
+    backend: 'aws',
+    region_name: 'eu-west-1',
+    provision_type: 'spot_low_price',
```

Comment:
spot

Reply:
It's AWS... Isn't that the default for AWS anymore ?

Reply:
no, we use spot

Reply:
it's working according to a different strategy now: it tries spot duration first, then try spot_low_proce and then on_demand if `instance_provision_fallback_on_demand` is set true in the YAML

Reply:
seem like the artifacts pipelines were missed to take this default.
fix in a new commit